tests/test_product_requirements_text_QCI97.py

```diff
@@ -0,0 +1,77 @@
+# QCI-97
+
+# In the "Select products to deploy screen, there is a text box to the right
+# of the products list.   It as you select various products will contain text
+# that explains what is required by each product selected.   Be sure the text
+# is correct for the products selected.
+
+def test_product_requirements_text(new_deployment_pg):
+    general_expected_text = (
+        "General\nRed Hat Customer Portal username and password. This account"
+        " must be an Organization Administrator within the Portal\nSufficient"
+        " subscriptions to complete this deployment\nHost machines are started"
+        " and discoverable\nNetworks and subsets are set up and available"
+    )
+    rhv_expected_text = "\n".join([
```

Comment:
A better way might be to put the text in a file and read it in....but I like this better than one one line for sure.